dev/tools/gen_defaults/lib/bottom_sheet_template.dart

```diff
@@ -30,6 +31,12 @@ class _${blockName}DefaultsM3 extends BottomSheetThemeData {
 
   @override
   Color? get shadowColor => Colors.transparent;
+
+  @override
+  Color? get dragHandleColor => ${componentColor("md.comp.sheet.bottom.docked.drag-handle")};
```

Comment:
This property and dragHandleSize are guaranteed to be non-null, so the overrides can return Color and Size respectively.